### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by pkrukowski1**
I propose the following changes:
- I have noticed that the weights of the classification head (local, linear, and RBF) were not copied from the old model, which is why the training was difficult.
- I modified the rbf.py file to prevent updating the mask when it is filled with ones.

### CODE REVIEW

**Review by OneAndZero24 — CHANGES_REQUESTED** [review]
One doubt. Other than that, great, you are right we forgot non incrementable params 😓

File: src/model/layer/rbf.py [hunk 1]
```diff
@@ -280,8 +280,11 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
         Returns:
             torch.Tensor: Output tensor of shape (batch_size, out_features).
         """
-        if self.growing_mask and self.training:
+
+        if self.growing_mask and self.training and not (self.mask == torch.ones_like(self.mask)).all():
```
[the remark this answers is not in the document]
Comment (pkrukowski1) [reply src/model/layer/rbf.py:284]: But we have 'not' in the condition, so the value False will be produced only when the mask is full of ones.

[the remark this answers is not in the document]
Comment (pkrukowski1) [reply src/model/layer/rbf.py:284]: But we have 'not' in the condition, so the value False will be produced only when the mask is full of ones.

[the remark this answers is not in the document]
Comment (OneAndZero24) [reply src/model/layer/rbf.py:284]: Right sorry 😅

**Review by OneAndZero24 — APPROVED** [review]
Great job
